app/models/champs/header_section_champ.rb

```diff
@@ -25,14 +25,6 @@ def search_terms
     # The user cannot enter any information here so it doesn’t make much sense to search
   end
 
-  def libelle_with_section_index
-    if sections&.none?(&:libelle_with_section_index?)
-      "#{section_index}. #{libelle}"
-    else
-      libelle
-    end
-  end
-
   def libelle_with_section_index?
     libelle =~ /^\d/
```

Comment:
```suggestion
    libelle =~ /^(\s*)?\d/
``` au cas ou il y ait un espace blanc en debut de titre ?

Reply:
J'ai prévu de normaliser tous les libellés